docs/react/toggling-devices.mdx

```diff
@@ -4,62 +4,61 @@ sidebar_position: 4
 
-Turning off the device saves battery and CPU power and turns off the LED.
-It takes longer to turn it back on, so other participants need to wait longer to receive the first data.
-You would use this method with a camera, as it's usually acceptable to wait longer to see someone's face.
-A disabled camera indicator reassures the user that the camera is truly turned off.
+## Turning Cameras On and Off - `toggleCamera()`
 
-Muting and unmuting is faster, but a muted device still uses resources.
-This is useful for microphones, as it is common to mute and unmute during a meeting.
-Unmuting needs to be quick to capture the first word of a sentence.
+This function controls the physical operational state of the camera.
 
-Both APIs are available for camera as well as for the microphone.
+- **Turning the camera off**: This action stops the camera device, disables the media stream, and pauses streaming. The webcam indicator light will shut down.
+- **Turning the camera on**: This action starts the camera and resumes streaming, allowing other participants to see video after a brief initialization period.
 
-## Muting and unmuting - `toggleMute()`
-
-The toggle mute function allows you to enable or disable the media stream without affecting the device's operational status.
-If the device is muted, it starts the device, enables the media stream, and starts (or resumes) streaming.
-When it's unmuted, it disables the media stream and pauses streaming but does not stop the device.
-In the case when the device is off, it starts the device, enables the media stream, and starts (or resumes) streaming.
-
-### Usage example
-
-Use this function to programmatically toggle the mute state based on user interactions or other application logic.
+### Usage Example
 
 ```tsx
 import { useCamera } from "@fishjam-cloud/react-client";
 
 export function CameraControl() {
-  // highlight-next-line
-  const { toggleMute } = useCamera();
+  const { toggleCamera } = useCamera();
 
-  return <button onClick={() => toggleMute()}>Toggle camera mute</button>;
+  return <button onClick={() => toggleCamera()}>Toggle Camera Power</button>;
 }
 ```
 
-This component provides a button that allows users to mute or unmute the camera stream.
+## Toggling Microphones - `toggleMicrophone()` and `toggleMicrophoneMute()`
+
+### Turning Microphones On and Off - `toggleMicrophone()`
 
-## Turning devices on and off - `toggleDevice()`
+This function toggles the microphone's physical operational state.
 
-The toggle device function control involves physically toggling the device's operational state.  
-If the device is off, it starts the device and begins (or resumes) streaming.  
-When it's on, it disables the media stream, pauses streaming, and stops the device.
+- **Turning the microphone off**: This stops the microphone, disables the media stream, and pauses any audio transmission.
+- **Turning the microphone on**: This enables the microphone and resumes audio streaming.
```

Comment:
**suggestion:** Maybe put some emphasis here that these functions can take a noticeable amount of time to take effect